product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -74,6 +74,7 @@ The Always-on architectures are built from either one group in a single location
 Tables are created across both groups, so any change goes to all nodes, not just to nodes in the local group.
 
 One node in each group is selected as the group's write leader. Proxies then direct application writes and queries to the write leader. The other nodes are replicas of the write leader. If, at any point, the write leader is seen to be unavailable, the remaining nodes in the group select a new write leader from the group the proxies direct traffic to that node. Scalability isn't the goal of this architecture.
```

Comment:
@jpe442 It looks like a word is missing here? Should it say "...and the proxies  direct traffic..."?